test/component/uk/gov/hmrc/individualsincomeapi/VersioningSpec.scala

```diff
@@ -54,6 +54,20 @@ class VersioningSpec extends BaseSpec {
       Json.parse(response.body) shouldBe validResponsePayload
     }
 
+    scenario("Requests with an accept header version P2.0") {
+      Given("A valid privileged Auth bearer token")
+      AuthStub.willAuthorizePrivilegedAuthToken(authToken, incomeScope)
+
+      When("A request to the match citizen endpoint is made with version P2.0 accept header")
+      val response = invokeWithHeaders(s"/sandbox?matchId=$sandboxMatchId", AUTHORIZATION -> authToken, acceptHeaderP2)
+
+      Then("The response status should be 200")
```

Comment:
Test specifies 200 then we expect INTERNAL_SERVER_ERROR?

Please correct all occurrences.